Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+lxml = "==4.2.4"
```

Comment:
Pinned dependency is good for me. The [changelog](https://lxml.de/4.2/changes-4.2.4.html) shows a certain activity.

Reply:
Pinning isn't a good idea for libraries, unless the dependency doesn't version correctly.

Reply:
Likewise libraries shouldn't have lock files.

Reply:
We can do `~= 4.2.4` here (`4.2.x`) or even `~= 4.2` (`4.x`), although it's `setup.py` that could actually enforce pinning (but it almost never does, deprecated/removed `dependency_links`). 

The lock file is the usual trade-off for the reproducible build, could use some form of the [`lowest`/`latest` double dependencies build](https://github.com/elifesciences/api-client-php/blob/master/project_tests.sh#L5-L10) to "unlock" it during CI in separate builds.

Reply:
I think we need the ``Pipfile`` in place of ``requirements.txt`` to easily set up the virtual environment, but I guess we could delete and gitignore the ``Pipfile.lock`` file if you think so. Those files won't be part of the distribution when you install it anyway, it will only be the contents of the ``/validator`` folder.